### PR TITLE
add celery related table only api-test stage

### PR DESCRIPTION
**Pull request by RichardXuan**
Another way to fix api-test pipeline, not influence the Django migrate.

### CODE REVIEW

File: openlcs/tasks/models.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class TaskMeta(models.Model):
     class Meta:
         db_table = 'celery_taskmeta'
         app_label = 'tasks'
+        managed = False
```
[the remark this answers is not in the document]
Comment (RichardXuan) [reply openlcs/tasks/models.py:19]: Thanks for your review

File: tests/functional/celery_meta.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    args bytea,
+    kwargs bytea,
+    worker character varying(155),
+    retries integer,
+    queue character varying(155)
+);
+ALTER TABLE ONLY celery_taskmeta
+    ADD CONSTRAINT celery_taskmeta_pkey PRIMARY KEY (id);
+ALTER TABLE ONLY celery_taskmeta
+    ADD CONSTRAINT celery_taskmeta_task_id_key UNIQUE (task_id);
```
Comment (hanchuntao) [inline tests/functional/celery_meta.sql:21]: Add blank line

File: tests/functional/celery_meta.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+CREATE SEQUENCE task_id_sequence
+               START WITH 1
+               INCREMENT BY 1
+               NO MINVALUE
+               NO MAXVALUE
+               CACHE 1;
+
+
```
Comment (hanchuntao) [inline tests/functional/celery_meta.sql:29]: Delete extra blank line

File: tests/functional/celery_meta.sql [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+CREATE TABLE celery_tasksetmeta (
+    id integer NOT NULL,
+    taskset_id character varying(155),
+    result bytea,
+    date_done timestamp without time zone
+);
```
Comment (hanchuntao) [inline tests/functional/celery_meta.sql:35]: Add blank lines

File: tests/functional/celery_meta.sql [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ALTER TABLE celery_tasksetmeta
+    ADD CONSTRAINT celery_tasksetmeta_pkey PRIMARY KEY (id);
+ALTER TABLE celery_tasksetmeta
+    ADD CONSTRAINT celery_tasksetmeta_taskset_id_key UNIQUE (taskset_id);
+CREATE SEQUENCE taskset_id_sequence
```
Comment (hanchuntao) [inline tests/functional/celery_meta.sql:40]: Add blank lines

**Pull request merged**